src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,54 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String SEPARATOR = ",";
+    private static final int INDEX_NAME = 0;
+    private static final int INDEX_AMOUNT = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
getStatistic should call three private methods, each dedicated to one job only, 1 - read from file, 2 - create report from that info, 3 - write report  to the file